README.md

```diff
@@ -5,27 +5,29 @@ This plugin adds `helm update-config` command to Helm CLI. It allows to update c
 ## Install
 
 ```
-helm plugin install https://github.com/burdiyan/helm-update-config
+helm plugin install https://github.com/bluebosh/helm-update-config
 ```
 
 ## Usage
 
 To change image tag of `smiling-penguin` release:
 
 ```
-helm update-config smiling-penguin --set=image.tag=stable
+helm update-config smiling-penguin --set-value=image.tag=stable
+helm update-config smiling-penguin --reset-value=true
+helm update-config smiling-penguin --value=file_path1
 ```
```

Comment:
Hi @EmilyEmily,

As we also support both --set-value and --value together, would you please also add this in usage, and we prefer the value in --set-value if there's a same one in --value file. Thank you!

Reply:
added samples.